Build failure: dooit

### Nixpkgs version

- Unstable (25.11)

### Steps to reproduce

Build dooit

### Can Hydra reproduce this build failure?

Yes, Hydra can reproduce this build failure.

### Link to Hydra build job

https://hydra.nixos.org/build/306063143/nixlog/1

### Relevant log output

```console
dooit> pytest flags: -m pytest
dooit> ============================= test session starts ==============================
dooit> platform linux -- Python 3.13.6, pytest-8.4.1, pluggy-1.6.0
dooit> rootdir: /build/source
dooit> configfile: pyproject.toml
dooit> testpaths: tests
dooit> plugins: Faker-37.5.3
dooit> collected 81 items
dooit>
dooit> tests/test_core/test_css_manager.py ..                                   [  2%]
dooit> tests/test_core/test_date_parse.py .                                     [  3%]
dooit> tests/test_core/test_hooks.py .                                          [  4%]
dooit> tests/test_core/test_model.py .....                                      [ 11%]
dooit> tests/test_core/test_todo.py .......................                     [ 39%]
dooit> tests/test_core/test_workspace.py ...........                            [ 53%]
dooit> tests/test_ui/test_bar.py F                                              [ 54%]
dooit> tests/test_ui/test_formatter.py FFFFF                                    [ 60%]
dooit> tests/test_ui/test_help_screen.py F                                      [ 61%]
dooit> tests/test_ui/test_key_resolve.py F                                      [ 62%]
dooit> tests/test_ui/test_loader.py F                                           [ 64%]
dooit> tests/test_ui/test_search.py F                                           [ 65%]
dooit> tests/test_ui/test_sort.py FFFFFF                                        [ 72%]
dooit> tests/test_ui/test_startup.py F                                          [ 74%]
dooit> tests/test_ui/test_todo_tree.py FFFFFFFFFFF                              [ 87%]
dooit> tests/test_ui/test_workspace_tree.py FFFFFFFFFF                          [100%]
dooit>
dooit> =================================== FAILURES ===================================
dooit> ______________________________ test_bar_switcher _______________________________
dooit> async def functions are not natively supported.
dooit> You need to install a suitable plugin for your async framework, for example:
dooit>   - anyio
dooit>   - pytest-asyncio
dooit>   - pytest-tornasync
dooit>   - pytest-trio
dooit>   - pytest-twisted
dooit> ______________________________ test_no_formatting ______________________________
dooit> async def functions are not natively supported.
dooit> You need to install a suitable plugin for your async framework, for example:
dooit>   - anyio
dooit>   - pytest-asyncio
dooit>   - pytest-tornasync
dooit>   - pytest-trio
dooit>   - pytest-twisted
dooit> ____________________________ test_basic_formatting _____________________________
dooit> async def functions are not natively supported.
dooit> You need to install a suitable plugin for your async framework, for example:
dooit>   - anyio
dooit>   - pytest-asyncio
dooit>   - pytest-tornasync
dooit>   - pytest-trio
dooit>   - pytest-twisted
dooit> ___________________________ test_multiple_formatting ___________________________
dooit> async def functions are not natively supported.
dooit> You need to install a suitable plugin for your async framework, for example:
dooit>   - anyio
dooit>   - pytest-asyncio
dooit>   - pytest-tornasync
dooit>   - pytest-trio
dooit>   - pytest-twisted
dooit> ________________________ test_multiple_formatting_skip _________________________
dooit> async def functions are not natively supported.
dooit> You need to install a suitable plugin for your async framework, for example:
dooit>   - anyio
dooit>   - pytest-asyncio
dooit>   - pytest-tornasync
dooit>   - pytest-trio
dooit>   - pytest-twisted
dooit> _______________________ test_multiple_formatting_toggle ________________________
dooit> async def functions are not natively supported.
dooit> You need to install a suitable plugin for your async framework, for example:
dooit>   - anyio
dooit>   - pytest-asyncio
dooit>   - pytest-tornasync
dooit>   - pytest-trio
dooit>   - pytest-twisted
dooit> ____________________________ test_help_screen_mount ____________________________
dooit> async def functions are not natively supported.
dooit> You need to install a suitable plugin for your async framework, for example:
dooit>   - anyio
dooit>   - pytest-asyncio
dooit>   - pytest-tornasync
dooit>   - pytest-trio
dooit>   - pytest-twisted
dooit> ____________________________ test_base_screen_keys _____________________________
dooit> async def functions are not natively supported.
dooit> You need to install a suitable plugin for your async framework, for example:
dooit>   - anyio
dooit>   - pytest-asyncio
dooit>   - pytest-tornasync
dooit>   - pytest-trio
dooit>   - pytest-twisted
dooit> _________________________________ test_loader __________________________________
dooit> async def functions are not natively supported.
dooit> You need to install a suitable plugin for your async framework, for example:
dooit>   - anyio
dooit>   - pytest-asyncio
dooit>   - pytest-tornasync
dooit>   - pytest-trio
dooit>   - pytest-twisted
dooit> _________________________________ test_search __________________________________
dooit> async def functions are not natively supported.
dooit> You need to install a suitable plugin for your async framework, for example:
dooit>   - anyio
dooit>   - pytest-asyncio
dooit>   - pytest-tornasync
dooit>   - pytest-trio
dooit>   - pytest-twisted
dooit> ___________________________ test_search_on_workspace ___________________________
dooit> async def functions are not natively supported.
dooit> You need to install a suitable plugin for your async framework, for example:
dooit>   - anyio
dooit>   - pytest-asyncio
dooit>   - pytest-tornasync
dooit>   - pytest-trio
dooit>   - pytest-twisted
dooit> _____________________________ test_search_on_todo ______________________________
dooit> async def functions are not natively supported.
dooit> You need to install a suitable plugin for your async framework, for example:
dooit>   - anyio
dooit>   - pytest-asyncio
dooit>   - pytest-tornasync
dooit>   - pytest-trio
dooit>   - pytest-twisted
dooit> ____________________________ test_options_highlight ____________________________
dooit> async def functions are not natively supported.
dooit> You need to install a suitable plugin for your async framework, for example:
dooit>   - anyio
dooit>   - pytest-asyncio
dooit>   - pytest-tornasync
dooit>   - pytest-trio
dooit>   - pytest-twisted
dooit> ______________________________ test_reverse_sort _______________________________
dooit> async def functions are not natively supported.
dooit> You need to install a suitable plugin for your async framework, for example:
dooit>   - anyio
dooit>   - pytest-asyncio
dooit>   - pytest-tornasync
dooit>   - pytest-trio
dooit>   - pytest-twisted
dooit> _____________________________ test_sort_cancelled ______________________________
dooit> async def functions are not natively supported.
dooit> You need to install a suitable plugin for your async framework, for example:
dooit>   - anyio
dooit>   - pytest-asyncio
dooit>   - pytest-tornasync
dooit>   - pytest-trio
dooit>   - pytest-twisted
dooit> ____________________________ test_description_sort _____________________________
dooit> async def functions are not natively supported.
dooit> You need to install a suitable plugin for your async framework, for example:
dooit>   - anyio
dooit>   - pytest-asyncio
dooit>   - pytest-tornasync
dooit>   - pytest-trio
dooit>   - pytest-twisted
dooit> _________________________________ test_startup _________________________________
dooit> async def functions are not natively supported.
dooit> You need to install a suitable plugin for your async framework, for example:
dooit>   - anyio
dooit>   - pytest-asyncio
dooit>   - pytest-tornasync
dooit>   - pytest-trio
dooit>   - pytest-twisted
dooit> _____________________________ test_todo_tree_focus _____________________________
dooit> async def functions are not natively supported.
dooit> You need to install a suitable plugin for your async framework, for example:
dooit>   - anyio
dooit>   - pytest-asyncio
dooit>   - pytest-tornasync
dooit>   - pytest-trio
dooit>   - pytest-twisted
dooit> _____________________________ test_todo_formatter ______________________________
dooit> async def functions are not natively supported.
dooit> You need to install a suitable plugin for your async framework, for example:
dooit>   - anyio
dooit>   - pytest-asyncio
dooit>   - pytest-tornasync
dooit>   - pytest-trio
dooit>   - pytest-twisted
dooit> ______________________________ test_no_node_error ______________________________
dooit> async def functions are not natively supported.
dooit> You need to install a suitable plugin for your async framework, for example:
dooit>   - anyio
dooit>   - pytest-asyncio
dooit>   - pytest-tornasync
dooit>   - pytest-trio
dooit>   - pytest-twisted
dooit> ____________________________ test_todo_tree_layout _____________________________
dooit> async def functions are not natively supported.
dooit> You need to install a suitable plugin for your async framework, for example:
dooit>   - anyio
dooit>   - pytest-asyncio
dooit>   - pytest-tornasync
dooit>   - pytest-trio
dooit>   - pytest-twisted
dooit> _____________________________ test_incorrect_edit ______________________________
dooit> async def functions are not natively supported.
dooit> You need to install a suitable plugin for your async framework, for example:
dooit>   - anyio
dooit>   - pytest-asyncio
dooit>   - pytest-tornasync
dooit>   - pytest-trio
dooit>   - pytest-twisted
dooit> _______________________________ test_remove_todo _______________________________
dooit> async def functions are not natively supported.
dooit> You need to install a suitable plugin for your async framework, for example:
dooit>   - anyio
dooit>   - pytest-asyncio
dooit>   - pytest-tornasync
dooit>   - pytest-trio
dooit>   - pytest-twisted
dooit> ___________________________________ test_due ___________________________________
dooit> async def functions are not natively supported.
dooit> You need to install a suitable plugin for your async framework, for example:
dooit>   - anyio
dooit>   - pytest-asyncio
dooit>   - pytest-tornasync
dooit>   - pytest-trio
dooit>   - pytest-twisted
dooit> _________________________________ test_urgency _________________________________
dooit> async def functions are not natively supported.
dooit> You need to install a suitable plugin for your async framework, for example:
dooit>   - anyio
dooit>   - pytest-asyncio
dooit>   - pytest-tornasync
dooit>   - pytest-trio
dooit>   - pytest-twisted
dooit> ______________________________ test_status_change ______________________________
dooit> async def functions are not natively supported.
dooit> You need to install a suitable plugin for your async framework, for example:
dooit>   - anyio
dooit>   - pytest-asyncio
dooit>   - pytest-tornasync
dooit>   - pytest-trio
dooit>   - pytest-twisted
dooit> ______________________________ test_effort_change ______________________________
dooit> async def functions are not natively supported.
dooit> You need to install a suitable plugin for your async framework, for example:
dooit>   - anyio
dooit>   - pytest-asyncio
dooit>   - pytest-tornasync
dooit>   - pytest-trio
dooit>   - pytest-twisted
dooit> ____________________________ test_recurrence_change ____________________________
dooit> async def functions are not natively supported.
dooit> You need to install a suitable plugin for your async framework, for example:
dooit>   - anyio
dooit>   - pytest-asyncio
dooit>   - pytest-tornasync
dooit>   - pytest-trio
dooit>   - pytest-twisted
dooit> _____________________________ test_workspaces_tree _____________________________
dooit> async def functions are not natively supported.
dooit> You need to install a suitable plugin for your async framework, for example:
dooit>   - anyio
dooit>   - pytest-asyncio
dooit>   - pytest-tornasync
dooit>   - pytest-trio
dooit>   - pytest-twisted
dooit> ______________________________ test_base_addition ______________________________
dooit> async def functions are not natively supported.
dooit> You need to install a suitable plugin for your async framework, for example:
dooit>   - anyio
dooit>   - pytest-asyncio
dooit>   - pytest-tornasync
dooit>   - pytest-trio
dooit>   - pytest-twisted
dooit> _______________________ test_workspace_remove_cancelled ________________________
dooit> async def functions are not natively supported.
dooit> You need to install a suitable plugin for your async framework, for example:
dooit>   - anyio
dooit>   - pytest-asyncio
dooit>   - pytest-tornasync
dooit>   - pytest-trio
dooit>   - pytest-twisted
dooit> ____________________________ test_workspace_remove _____________________________
dooit> async def functions are not natively supported.
dooit> You need to install a suitable plugin for your async framework, for example:
dooit>   - anyio
dooit>   - pytest-asyncio
dooit>   - pytest-tornasync
dooit>   - pytest-trio
dooit>   - pytest-twisted
dooit> ______________________________ test_no_node_error ______________________________
dooit> async def functions are not natively supported.
dooit> You need to install a suitable plugin for your async framework, for example:
dooit>   - anyio
dooit>   - pytest-asyncio
dooit>   - pytest-tornasync
dooit>   - pytest-trio
dooit>   - pytest-twisted
dooit> ___________________________ test_shifts_single_item ____________________________
dooit> async def functions are not natively supported.
dooit> You need to install a suitable plugin for your async framework, for example:
dooit>   - anyio
dooit>   - pytest-asyncio
dooit>   - pytest-tornasync
dooit>   - pytest-trio
dooit>   - pytest-twisted
dooit> _________________________________ test_shifts __________________________________
dooit> async def functions are not natively supported.
dooit> You need to install a suitable plugin for your async framework, for example:
dooit>   - anyio
dooit>   - pytest-asyncio
dooit>   - pytest-tornasync
dooit>   - pytest-trio
dooit>   - pytest-twisted
dooit> ____________________________ test_cursor_movements _____________________________
dooit> async def functions are not natively supported.
dooit> You need to install a suitable plugin for your async framework, for example:
dooit>   - anyio
dooit>   - pytest-asyncio
dooit>   - pytest-tornasync
dooit>   - pytest-trio
dooit>   - pytest-twisted
dooit> ________________________ test_add_sibling_while_editing ________________________
dooit> async def functions are not natively supported.
dooit> You need to install a suitable plugin for your async framework, for example:
dooit>   - anyio
dooit>   - pytest-asyncio
dooit>   - pytest-tornasync
dooit>   - pytest-trio
dooit>   - pytest-twisted
dooit> ________________________ test_yank_and_paste_workspace _________________________
dooit> async def functions are not natively supported.
dooit> You need to install a suitable plugin for your async framework, for example:
dooit>   - anyio
dooit>   - pytest-asyncio
dooit>   - pytest-tornasync
dooit>   - pytest-trio
dooit>   - pytest-twisted
dooit> =============================== warnings summary ===============================
dooit> ../../nix/store/l9wxarvas1jpan9s1l0k1a5mx05g3qj6-python3.13-pytest-8.4.1/lib/python3.13/site-packages/_pytest/config/__init__.py:1474
dooit>   /nix/store/l9wxarvas1jpan9s1l0k1a5mx05g3qj6-python3.13-pytest-8.4.1/lib/python3.13/site-packages/_pytest/config/__init__.py:1474: PytestConfigWarning: Unknown config option: asyncio_default_fixture_loop_scope
dooit>
dooit>     self._warn_or_fail_if_strict(f"Unknown config option: {key}\n")
dooit>
dooit> ../../nix/store/l9wxarvas1jpan9s1l0k1a5mx05g3qj6-python3.13-pytest-8.4.1/lib/python3.13/site-packages/_pytest/config/__init__.py:1474
dooit>   /nix/store/l9wxarvas1jpan9s1l0k1a5mx05g3qj6-python3.13-pytest-8.4.1/lib/python3.13/site-packages/_pytest/config/__init__.py:1474: PytestConfigWarning: Unknown config option: asyncio_mode
dooit>
dooit>     self._warn_or_fail_if_strict(f"Unknown config option: {key}\n")
dooit>
dooit> -- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
dooit> =========================== short test summary info ============================
dooit> FAILED tests/test_ui/test_bar.py::test_bar_switcher - Failed: async def functions are not natively supported.
dooit> FAILED tests/test_ui/test_formatter.py::test_no_formatting - Failed: async def functions are not natively supported.
dooit> FAILED tests/test_ui/test_formatter.py::test_basic_formatting - Failed: async def functions are not natively supported.
dooit> FAILED tests/test_ui/test_formatter.py::test_multiple_formatting - Failed: async def functions are not natively supported.
dooit> FAILED tests/test_ui/test_formatter.py::test_multiple_formatting_skip - Failed: async def functions are not natively supported.
dooit> FAILED tests/test_ui/test_formatter.py::test_multiple_formatting_toggle - Failed: async def functions are not natively supported.
dooit> FAILED tests/test_ui/test_help_screen.py::test_help_screen_mount - Failed: async def functions are not natively supported.
dooit> FAILED tests/test_ui/test_key_resolve.py::test_base_screen_keys - Failed: async def functions are not natively supported.
dooit> FAILED tests/test_ui/test_loader.py::test_loader - Failed: async def functions are not natively supported.
dooit> FAILED tests/test_ui/test_search.py::test_search - Failed: async def functions are not natively supported.
dooit> FAILED tests/test_ui/test_sort.py::test_search_on_workspace - Failed: async def functions are not natively supported.
dooit> FAILED tests/test_ui/test_sort.py::test_search_on_todo - Failed: async def functions are not natively supported.
dooit> FAILED tests/test_ui/test_sort.py::test_options_highlight - Failed: async def functions are not natively supported.
dooit> FAILED tests/test_ui/test_sort.py::test_reverse_sort - Failed: async def functions are not natively supported.
dooit> FAILED tests/test_ui/test_sort.py::test_sort_cancelled - Failed: async def functions are not natively supported.
dooit> FAILED tests/test_ui/test_sort.py::test_description_sort - Failed: async def functions are not natively supported.
dooit> FAILED tests/test_ui/test_startup.py::test_startup - Failed: async def functions are not natively supported.
dooit> FAILED tests/test_ui/test_todo_tree.py::test_todo_tree_focus - Failed: async def functions are not natively supported.
dooit> FAILED tests/test_ui/test_todo_tree.py::test_todo_formatter - Failed: async def functions are not natively supported.
dooit> FAILED tests/test_ui/test_todo_tree.py::test_no_node_error - Failed: async def functions are not natively supported.
dooit> FAILED tests/test_ui/test_todo_tree.py::test_todo_tree_layout - Failed: async def functions are not natively supported.
dooit> FAILED tests/test_ui/test_todo_tree.py::test_incorrect_edit - Failed: async def functions are not natively supported.
dooit> FAILED tests/test_ui/test_todo_tree.py::test_remove_todo - Failed: async def functions are not natively supported.
dooit> FAILED tests/test_ui/test_todo_tree.py::test_due - Failed: async def functions are not natively supported.
dooit> FAILED tests/test_ui/test_todo_tree.py::test_urgency - Failed: async def functions are not natively supported.
dooit> FAILED tests/test_ui/test_todo_tree.py::test_status_change - Failed: async def functions are not natively supported.
dooit> FAILED tests/test_ui/test_todo_tree.py::test_effort_change - Failed: async def functions are not natively supported.
dooit> FAILED tests/test_ui/test_todo_tree.py::test_recurrence_change - Failed: async def functions are not natively supported.
dooit> FAILED tests/test_ui/test_workspace_tree.py::test_workspaces_tree - Failed: async def functions are not natively supported.
dooit> FAILED tests/test_ui/test_workspace_tree.py::test_base_addition - Failed: async def functions are not natively supported.
dooit> FAILED tests/test_ui/test_workspace_tree.py::test_workspace_remove_cancelled - Failed: async def functions are not natively supported.
dooit> FAILED tests/test_ui/test_workspace_tree.py::test_workspace_remove - Failed: async def functions are not natively supported.
dooit> FAILED tests/test_ui/test_workspace_tree.py::test_no_node_error - Failed: async def functions are not natively supported.
dooit> FAILED tests/test_ui/test_workspace_tree.py::test_shifts_single_item - Failed: async def functions are not natively supported.
dooit> FAILED tests/test_ui/test_workspace_tree.py::test_shifts - Failed: async def functions are not natively supported.
dooit> FAILED tests/test_ui/test_workspace_tree.py::test_cursor_movements - Failed: async def functions are not natively supported.
dooit> FAILED tests/test_ui/test_workspace_tree.py::test_add_sibling_while_editing - Failed: async def functions are not natively supported.
dooit> FAILED tests/test_ui/test_workspace_tree.py::test_yank_and_paste_workspace - Failed: async def functions are not natively supported.
dooit> ================== 38 failed, 43 passed, 2 warnings in 1.96s ===================
```

### Additional context

_No response_

### System metadata

nix-shell -p nix-info --run "nix-info -m"
 - system: `"x86_64-linux"`
 - host os: `Linux 6.16.1, NixOS, 25.11 (Xantusia), 25.11.20250819.2007595`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.30.2`
 - channels(root): `""`
 - nixpkgs: `/nix/store/pxnx7dbr0pz3qpgw2r315wdbb00afdw7-source`

### Notify maintainers

@kraanzu @wesleyjrz 

---

**Note for maintainers:** Please tag this issue in your pull request description. (i.e. `Resolves #ISSUE`.)


### I assert that this issue is relevant for Nixpkgs

- [x] I assert that this is a bug and not a support request.
- [x] I assert that this is not a [duplicate of an existing issue](https://github.com/NixOS/nixpkgs/issues?q=is%3Aopen+is%3Aissue+label%3A%220.kind%3A+build+failure%22). 
- [x] I assert that I have read the [NixOS Code of Conduct](https://github.com/NixOS/.github/blob/master/CODE_OF_CONDUCT.md) and agree to abide by it.

### Is this issue important to you?

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

--- a/pkgs/by-name/do/dooit/package.nix
+++ b/pkgs/by-name/do/dooit/package.nix
@@ -46,9 +46,10 @@ python3.pkgs.buildPythonApplication rec {
     export HOME=$(mktemp -d)
   '';
 
-  checkInputs = with python3.pkgs; [
+  nativeCheckInputs = with python3.pkgs; [
     pytestCheckHook
     faker
+    pytest-asyncio
   ];
 
   passthru = {

